awx/main/dispatch/worker/callback.py

```diff
@@ -151,12 +151,14 @@ def flush(self, force=False):
             metrics_singular_events_saved = 0
             metrics_events_batch_save_errors = 0
             metrics_events_broadcast = 0
+            metrics_job_lag = 0
             for cls, events in self.buff.items():
                 logger.debug(f'{cls.__name__}.objects.bulk_create({len(events)})')
                 for e in events:
                     if not e.created:
                         e.created = now
                     e.modified = now
+                    # metrics_job_lag = e.modified - e.created  # get the delta between modified and created. TODO: currently being overwritten
```

Comment:
we're going to want to `+=` add to `metrics_total_job_event_processing_seconds` here -- may need wrangle datetime timedelta to seconds in float